robot/Cumulus/resources/AccountPageObject.py

```diff
@@ -1,19 +1,17 @@
 from cumulusci.robotframework.pageobjects import ListingPage
 from cumulusci.robotframework.pageobjects import DetailPage
 from cumulusci.robotframework.pageobjects import pageobject
+from Basenpspobjects import BaseNPSPPage
 from NPSP import npsp_lex_locators
 
 @pageobject("Listing", "Account")
-class AccountListingPage(ListingPage):
+class AccountListingPage(ListingPage, BaseNPSPPage):
```

Comment:
I really apologize for this, but I gave a bad example. It would be better to change the order of these two. It doesn't actually affect your code in this specific case, but I know people may look at your code as an example and want to copy it. It would be better if we set a good example here.

I recommend  you change this line to look like this:

`class AccountingListingPage(BaseNPSPPage, ListingPage)`

Here's a link to a question on stackoverflow that answers the question of "why". Again, I apologize!

Reply:
@boakley I made updates. I dont see the link here can u share it with me again plz

Reply:
@skristem Are you planning to perform the changes as Bryan mentioned above in this PR?

Reply:
yes made the change already